### PR TITLE
docs(prebuilt): intro and platform docs for HMSPrebuilt

### PR DESCRIPTION
**Pull request by akankshat100ms**
- updated Prebuilt landing page (edits pending : image, 
- new files added for HMSPrebuilt component platform docs

### CODE REVIEW

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 1]
```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
-```
\ No newline at end of file
+```
+
+</StepsContainer>
+
+
+
+## Embed as Component
+100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing HMSRoomKit in your apps.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:86]:
```suggestion
100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing 100ms Room Kit package in your app.
```

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 2]
```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
+
+
+<FlexContainer>
+<Callout icon="link" title="HMSPrebuilt Component Basics">
+
+    - **HMSPrebuilt -** high-order component from HMSRoom Kit that handles all audio/video functionality within a room
+        - has default colors and styles of 100ms, which can be overridden with code.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:93]:
```suggestion
        - has default colors and styles of 100ms, which can be overridden from dashboard.
```

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 3]
```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
+        - HMSPrebuilt requires room-code for peers to join the room
+
+    - **HMSRoomKit** -  Collection of embeddable 100ms UI components for your apps.
+        - It is the first embeddable UI component of HMSRoomKit.
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:97]:
```suggestion
        - `HMSPrebuilt` is the first embeddable UI component of HMSRoomKit.
```

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 4]
```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
+<StepsContainer id="implement-breakout-room">
+
+### Create and Get room code
+HMSPrebuilt uses [room code](/concepts/v2/concepts/rooms/room-codes/room-codes) to get initialised, it allow users to uniquely join a `room_id` with a specifc `role`. Head over to 
+- [Create Room Codes](/docs/get-started/v2/get-started/prebuilt/room-codes/overview#creating-room-codes)
+- [Get Room Code](/docs/server-side/v2/api-reference/room-codes/create-room-code-api)
+
+
+### Embedding HMSPrebuilt component
+    - Import - HMSPrebuilt component requires importing HMSRoomKit
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:112]:
```suggestion
    - Import - HMSPrebuilt component requires adding & importing HMSRoomKit package
```

File: public/docs/v2/testPrebuilt.png
Comment (saikatmitra91) [inline public/docs/v2/testPrebuilt.png:1]: lets update this screenshot to match the current state of prebuilt component UI

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/ios/v2/quickstart/prebuilt-iOS.mdx [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (saikatmitra91) [inline docs/ios/v2/quickstart/prebuilt-iOS.mdx:1]: Lets have coming soon for iOS 

iOS might not be able to join in first release.

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 6]
```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
...
+  val debugInfo: Bool = false, // Optional debug info. Not needed for production apps
+}
+
+
+```
+</Tab>
+
+<Tab id='sample-code-2'>
+
+```swift section=SampleCode  tab=iOS
```
Comment (saikatmitra91) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:181]: lets have coming soon tab for iOS or lets not mention at all for iOS for first release

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/get-started/v2/get-started/prebuilt/prebuilt.mdx [hunk 7]
```diff
@@ -3,29 +3,48 @@ title: 100ms Prebuilt
 nav: 4.3
 ---
 
-100ms Prebuilt is a feature rich video conferencing / live streaming app built on top on our [sample web app](https://github.com/100mslive/100ms-web/). Share Prebuilt room links with users or [embed it as an iFrame](/get-started/v2/get-started/prebuilt/prebuilt#share-or-embed-as-iframe) in your application.
 
-Getting started with 100ms Prebuilt Room Links is the fastest way to test and go live with 100ms. [Test Prebuilt Link→](https://examples-videoconferencing.app.100ms.live/meeting/tmz-qdi-cyp)
+100ms Prebuilt is a feature-rich video conferencing/live streaming app built on top of our [sample web app](https://github.com/100mslive/100ms-web/). Prebuilt allows users to join sharable room links via desktop, meeting invites, or as an embeddable experience using an [iFrame](/docs/get-started/v2/get-started/prebuilt/prebuilt#share-or-embed-as-iframe), or as a native component in your apps.
```
Comment (PratimMallick) [inline docs/get-started/v2/get-started/prebuilt/prebuilt.mdx:7]: Is it built on top of web app? OR web app is built using the preview components

**Pull request merged**